createdisk-library.sh

```diff
@@ -388,7 +388,7 @@ function download_podman() {
     local arch=$2
 
     mkdir -p podman-remote/linux
-    curl -L https://github.com/containers/podman/releases/download/v4.1.1/podman-remote-static.tar.gz | tar -zx -C podman-remote/linux podman-remote-static
+    curl -L https://github.com/containers/podman/releases/download/v${version}/podman-remote-static.tar.gz | tar -zx -C podman-remote/linux podman-remote-static
```

Comment:
CI is failing because https://github.com/containers/podman/releases/v4.6.1/ have linux bits as per the arch. https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/crc-org_snc/879/pull-ci-crc-org-snc-release-4.15-e2e-microshift/1786112647393972224 


Also better to revert f4d7855ff6cb452b9d8d4d3ee3bce8d27eaf7a51 one and then add commit to fix the arch info

Reply:
It appears that they changed the directory structure in the archive as well (binary is now inside `bin` directory). Should the command to rename the binary keep it in the same directory or move it to `podman-remote/linux`?

Reply:
> It appears that they changed the directory structure in the archive as well (binary is now inside bin directory). 

If you look at the git history/git blame, there were several changes related to the podman archive structure in the past.

> Should the command to rename the binary keep it in the same directory or move it to podman-remote/linux?

I'd keep the destination file in this rename as other part of snc/crc might rely on it
```
createdisk-library.sh:    mv podman-remote/linux/podman-remote-static podman-remote/linux/podman-remote
```

Reply:
Thanks for the input Christophe.